spring-cloud-task-core/src/main/java/org/springframework/cloud/task/repository/dao/JdbcTaskExecutionDao.java

```diff
@@ -52,15 +53,16 @@
 public class JdbcTaskExecutionDao implements TaskExecutionDao {
 
 
-	public static String SELECT_CLAUSE = "TASK_EXECUTION_ID, "
+	public static String SELECT_CLAUSE = "TASK_EXECUTION_ID, TASK_EXTERNAL_EXECUTION_ID, "
```

Comment:
Being nit picky here but we probably don't need to prefix the name of the column with task...